Dockerfile

```diff
@@ -9,6 +9,4 @@ RUN pip install pipenv && \
     pipenv install --system && \
     pipenv sync --dev
 
-ENV PATH /root/.local/share/virtualenvs/genyrator-oIyKmRQj/bin:$PATH
```

Comment:
I need this to get pycharm to run anything which is not `python`. is there a reason to remove it?

Reply:
I'm pretty confident this can be removed. It is unrelated though so I'll move it into a separate PR.

Reply:
See #38